server/libs/infra/src/db/1675668246380-AddUpdatedAtColumnToAlbums1.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddUpdatedAtColumnToAlbums11675668246380 implements MigrationInterface {
```

Comment:
I think this file is not needed? (only an artifact of generating the other migration file)